primitives/src/v2/mod.rs

```diff
@@ -1394,9 +1422,11 @@ pub type CheckedMultiDisputeStatementSet = Vec<CheckedDisputeStatementSet>;
 #[derive(Encode, Decode, Clone, RuntimeDebug, PartialEq, TypeInfo)]
 pub struct DisputeState<N = BlockNumber> {
 	/// A bitfield indicating all validators for the candidate.
-	pub validators_for: BitVec<u8, bitvec::order::Lsb0>, // one bit per validator.
+	pub validators_for: BitVec<u8, bitvec::order::Lsb0>,
+	// one bit per validator.
```

Comment:
please revert unrelated formatting changes, we're using a specific nightly version for fmt